### PR TITLE
S11t6

### PR DESCRIPTION
**Pull request by mariagus**
Add a name search

### CODE REVIEW

File: src/Controllers/FrontEnd/ApplicantsPageController.php [hunk 1]
```diff
@@ -42,6 +42,7 @@ public function __construct(PhpRenderer $renderer, StageModel $stageModel, Appli
     public function __invoke(Request $request, Response $response, array $args)
     {
         if ($_SESSION['loggedIn'] === true) {
+            $_SESSION['name'] = $request->getQueryParams()['name'] ?? $_SESSION['name'] ?? '%';
```
Comment (imypullan) [inline src/Controllers/FrontEnd/ApplicantsPageController.php:45]: all looking good on this page

File: src/Factories/Controllers/API/GetApplicantByNameControllerFactory.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
```
Comment (imypullan) [inline src/Factories/Controllers/API/GetApplicantByNameControllerFactory.php:1]: this file can be deleted right?

File: src/Factories/Controllers/API/GetApplicationFormControllerFactory.php [hunk 3]
```diff
@@ -18,6 +18,7 @@ class GetApplicationFormControllerFactory
     public function __invoke(ContainerInterface $container)
```
Comment (imypullan) [inline src/Factories/Controllers/API/GetApplicationFormControllerFactory.php:18]: and this file can also be deleted?

File: templates/applicants.phtml [hunk 4]
```diff
@@ -29,6 +29,13 @@
                         <a href="#apprentice" aria-controls="apprentice" role="tab" data-toggle="tab">
                             Apprentice Applicants
                         </a>
+                    </li>
+                    <li class="pull-right">
+                        <label for="name">
+                            Search Name
+                            <input class="submitApplicant form-control" type="text" name="name" id="name" value=""/>
```
Comment (imypullan) [inline templates/applicants.phtml:36]: does this need the id? is that for styling?

Comment (mariagus) [reply templates/applicants.phtml:36]: this shouldn't even be in the repo! haha

**Pull request merged**